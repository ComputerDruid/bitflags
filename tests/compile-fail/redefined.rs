#[macro_use]
extern crate bitflags;

bitflags! {
    pub struct Flags1 {
        const A = 1;
    }
}

bitflags! {
    pub struct Flags1 {
        const A = 1;
    }
}
