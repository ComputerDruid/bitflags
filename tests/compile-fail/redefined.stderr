error: no rules expected the token `{`
 --> tests/compile-fail/redefined.rs:5:23
  |
5 |     pub struct Flags1 {
  |                       ^ no rules expected this token in macro call

error: no rules expected the token `{`
  --> tests/compile-fail/redefined.rs:11:23
   |
11 |     pub struct Flags1 {
   |                       ^ no rules expected this token in macro call

error[E0601]: `main` function not found in crate `$CRATE`
  --> tests/compile-fail/redefined.rs:14:2
   |
14 | }
   |  ^ consider adding a `main` function to `$DIR/tests/compile-fail/redefined.rs`
